sdk/spring/spring-cloud-azure-autoconfigure/src/test/java/com/azure/spring/cloud/autoconfigure/storage/AzureStorageAutoConfigurationTests.java

```diff
@@ -21,8 +31,8 @@ class AzureStorageAutoConfigurationTests {
     private static final String STORAGE_CONNECTION_STRING_PATTERN = "DefaultEndpointsProtocol=https;AccountName=%s;AccountKey=%s;EndpointSuffix=core.windows.net";
     private final ApplicationContextRunner contextRunner = new ApplicationContextRunner()
         .withConfiguration(AutoConfigurations.of(AzureStorageBlobAutoConfiguration.class))
-        .withConfiguration(AutoConfigurations.of(AzureStorageFileShareAutoConfiguration.class));
-
+        .withConfiguration(AutoConfigurations.of(AzureStorageFileShareAutoConfiguration.class))
+        .withConfiguration(AutoConfigurations.of(AzureStorageQueueAutoConfiguration.class));
```

Comment:
I think this class should by default with none Autoconfiguration classes, we should specify them in each test case.